src/ui.ts

```diff
@@ -41,6 +42,8 @@ export type ConsentManagerAPI = {
   toggleConsentManager(options?: ShowConsentManagerOptions): Promise<void>;
   /** Hide consent manager */
   hideConsentManager(): Promise<void>;
+  /** Opt out of the sale of personal info & show a disclosure */
+  doNotSell(auth: AirgapAuth): Promise<void>;
```

Comment:
```suggestion
  doNotSell(auth: AirgapAuth, options?: ShowConsentManagerOptions): Promise<void>;
```

would maybe add this as well for now, or put auth inside the `ShowConsentManagerOptions` object. just thinking if we ever offered multiple do not sell interfaces, it would be nice to allow flexibility here?

Reply:
Good idea, thanks!

Reply:
I like having auth as the first argument so you can pass our utility directly to addEventListener (e.g. `optOutButton.addEventListener('click', transcend.doNotSell, {once: true})`)

Reply:
We have to be careful to not use any primitive values for the second+ arguments, as that can be pre-filled by the event listener/caller in some scenarios. Having it as an object will satisfy that constraint.